### ISSUE TITLE
join() argument must be str, bytes, or os.PathLike object, not 'NoneType'

### ISSUE THREAD

**Issue by pegasusearl** [lutris/lutris#4147]
```
OS: Manjaro Linux x86_64 
Kernel: 5.15.25-1-MANJARO 
Uptime: 17 mins 
Packages: 1632 (pacman), 8 (snap) 
Shell: zsh 5.8.1 
Resolution: 1920x1080, 1920x1080 
DE: GNOME 41.4 
WM: Mutter 
WM Theme: Matcha-light-sea 
Theme: Vertex-Dark [GTK2/3] 
Icons: Fluent-green-dark [GTK2/3] 
Terminal: gnome-terminal 
CPU: AMD Ryzen 5 3500 (6) @ 3.600GHz 
GPU: NVIDIA GeForce GTX 1650 
Memory: 3940MiB / 15921MiB 

```

Running Lutris for the first time.
Took very long time to load and in the end this popup showed up.
![13_05-02-40_299](https://user-images.githubusercontent.com/23167087/158036658-f1265ac3-d4e1-41dd-9b47-01631794aebd.png)

```
lutris                                                                                        INT ✘  1m 54s  
2022-03-13 05:01:27,896: Starting Lutris 0.5.9.1
2022-03-13 05:01:27,900: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,901: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,901: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,901: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,901: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,902: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,902: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,902: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,902: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,904: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,904: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,904: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,905: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,905: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,905: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,905: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,905: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,905: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,914: No cores found
2022-03-13 05:01:27,926: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,926: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,927: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of VKD3D versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of D3D Extras versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:27,929: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:38,007: Failed to get runtimes: Unable to connect to server https://lutris.net/api/runtimes?enabled=1: <urlopen error [Errno -3] Temporary failure in name resolution>
2022-03-13 05:01:48,023: Failed to get url https://api.github.com/repos/lutris/dxvk/releases: Unable to connect to server https://api.github.com/repos/lutris/dxvk/releases: <urlopen error [Errno -3] Temporary failure in name resolution>
2022-03-13 05:01:48,024: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:48,025: Loading of DXVK versions failed, defaulting to locally available versions
2022-03-13 05:01:58,041: Failed to get url https://api.github.com/repos/lutris/dxvk-nvapi/releases: Unable to connect to server https://api.github.com/repos/lutris/dxvk-nvapi/releases: <urlopen error [Errno -3] Temporary failure in name resolution>
2022-03-13 05:01:58,042: Loading of DXVK-NVAPI versions failed, defaulting to locally available versions
2022-03-13 05:01:58,043: Error while completing task <bound method LutrisInitDialog.initialize of <dialogs.LutrisInitDialog object at 0x7fed1d3d15c0 (lutris+gui+dialogs+LutrisInitDialog at 0x556f7fc80340)>>: <class 'TypeError'> join() argument must be str, bytes, or os.PathLike object, not 'NoneType'
  File "/usr/lib/python3.10/site-packages/lutris/util/jobs.py", line 34, in target
    result = self.function(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/lutris/gui/dialogs/__init__.py", line 190, in initialize
    init_lutris()
  File "/usr/lib/python3.10/site-packages/lutris/startup.py", line 187, in update_runtime
    dll_manager.upgrade()
  File "/usr/lib/python3.10/site-packages/lutris/util/wine/dll_manager.py", line 190, in upgrade
    if not self.is_available():
  File "/usr/lib/python3.10/site-packages/lutris/util/wine/dll_manager.py", line 86, in is_available
    return system.path_exists(self.path)
  File "/usr/lib/python3.10/site-packages/lutris/util/wine/dll_manager.py", line 53, in path
    return os.path.join(self.base_dir, self.version)
  File "/usr/lib/python3.10/posixpath.py", line 90, in join
    genericpath._check_arg_types('join', a, *p)
  File "/usr/lib/python3.10/genericpath.py", line 152, in _check_arg_types
    raise TypeError(f'{funcname}() argument must be str, bytes, or '


```

While I can just ignore it, I'm not sure how it affect the program. However there is another issue that I'm not sure if it's related.
I cannot install a game from lutris install script. When I pressed install on the web, the Lutris did not react. https://github.com/lutris/lutris/issues/4148

**Comment by rachmadaniHaryono** [comment]
this should be fixed by https://github.com/lutris/lutris/commit/e1e47d234c3f8148f72fb1c02b45c2774d7b7532

**Comment by danieljohnson2** [comment]
This looks like the issue we had when there were DNS problems while trying to update the runtime. I don't think anybody filed a bug, but it was fixed by PR #4034. Try the beta; should already be better there.

**Comment by pegasusearl** [comment]
Oh thats right. I'm sorry, I should have tried latest version before posting issues.
